lib/features/agenda/agenda_actions.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:pass_emploi_app/models/agenda.dart';
+
+class AgendaRequestAction {
+  final DateTime maintenant;
+
+  AgendaRequestAction(this.maintenant);
+}
+
+class AgendaRequestFailureAction {}
+
+class AgendaRequestSuccessAction {
+  final Agenda agenda;
+
+  AgendaRequestSuccessAction(this.agenda);
+}
```

Comment:
On a un state "Not Init", ca va etre plus coherent si on rajoute une action "Reset" ou "Not Init"